ui/app/helpers/mountable-secret-engines.js

```diff
@@ -143,15 +143,15 @@ export function wifEngines() {
 
 // The UI only supports configuration views for these secrets engines. The CLI must be used to manage other engine resources (i.e. roles, credentials).
 // Will eventually include gcp.
```

Comment:
Are you waiting to remove this comment when the configuration is added? Since you already added `gcp` below I wasn't sure!
```suggestion
```